### PR TITLE
USHIFT-2459: Add docs to namespaceOwnership

### PR DESCRIPTION
**Pull request by pacevedom**
**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

File: pkg/config/ingress.go [hunk 1]
```diff
@@ -14,6 +14,17 @@ type IngressConfig struct {
 }
 
 type RouteAdmissionPolicy struct {
+	// namespaceOwnership describes how host name claims across namespaces should
```
Comment (pmtk) [inline pkg/config/ingress.go:17]:
```suggestion
	// NamespaceOwnership describes how host name claims across namespaces should
```

Comment (pacevedom) [reply pkg/config/ingress.go]: There is this discrepancy, but I saw it in other parameters. Let me rephrase it to avoid mentioning it (its redundant anyway), as this is also quite common in other parameters.

File: pkg/config/ingress.go [hunk 2]
```diff
@@ -14,6 +14,16 @@ type IngressConfig struct {
 }
 
 type RouteAdmissionPolicy struct {
+	// Describe how host name claims across namespaces should be handled.
```
Comment (dhellmann) [inline pkg/config/ingress.go:17]:
```suggestion
	// Describes how host name claims across namespaces should be handled.
```

Comment (pacevedom) [reply pkg/config/ingress.go]: Done
